tests/test_x509.py

```diff
@@ -856,6 +856,38 @@ def test_add_unsupported_extension(self, backend):
                 critical=False,
             )
 
+    def test_add_unsupported_extension_via_init(self, backend):
+        private_key = RSA_KEY_2048.private_key(backend)
+
+        builder = x509.CertificateSigningRequestBuilder(
+            subject_name=x509.Name([
+                x509.NameAttribute(x509.OID_COUNTRY_NAME, u'US'),
+            ]),
+            extensions=[
+                x509.Extension(
+                    x509.OID_AUTHORITY_KEY_IDENTIFIER,
+                    False,
+                    x509.AuthorityKeyIdentifier('keyid', None, None)
+                )
+            ]
+        )
```

Comment:
If you have a better way to trigger the underlying error in the backend I'm happy to change it. If I don't do this I don't have a way to trigger the NotImplementedError in the backend since it will catch it inside the builder before it ever gets that far.